libs/gltfio/include/gltfio/SimpleViewer.h

```diff
@@ -461,8 +462,14 @@ void SimpleViewer::updateUserInterface() {
     if (ImGui::CollapsingHeader("View")) {
         ImGui::Indent();
         ImGui::Checkbox("Dithering", &mEnableDithering);
-        ImGui::Checkbox("FXAA", &mEnableFxaa);
         ImGui::Checkbox("MSAA 4x", &mEnableMsaa);
+        ImGui::Checkbox("TAA", &mTAAOptions.enabled);
+        // this clutters the UI and isn't that useful (except when working on TAA)
```

Comment:
`#ifndef NDEBUG` maybe?

Reply:
I really didn't think they were that useful other than when debugging TAA, I don't think people will tweak them in practice.